lib/members.rb

```diff
@@ -11,7 +11,9 @@ def initialize(url)
   end
 
   def to_a
```

Comment:
I think you should fix up some of the issues with the existing scraper first — e.g. replacing this `to_a` with a `members` field on a serialized MembersPage class — then come back to making this PR.